controller/space_iterations_test.go

```diff
@@ -301,7 +298,7 @@ func (rest *TestSpaceIterationREST) TestWICountsWithIterationListBySpace() {
 		Name:       "Sprint 2.1",
 		SpaceID:    spaceInstance.ID,
 		Path:       append(iteration2.Path, iteration2.ID),
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c